wgsl/index.bs

```diff
@@ -4511,6 +4511,9 @@ The <dfn noexport>address-of</dfn> operator converts a reference to its correspo
            If |r| is an [=invalid memory reference=], then the resulting
            pointer is also an invalid memory reference.
 
+           It is a [=shader-creation error=] if |T| is type stored in the
```

Comment:
Isn't it simpler to say |SC| must not be `handle` ?

And in https://gpuweb.github.io/gpuweb/wgsl/#module-scope-variables it says 

> If the store type is a texture type or a sampler type, then the variable declaration must not have a storage class decoration. The storage class will always be handle.


(In any case, I think the proposed text is missing "a", as in  "is a type stored in the...")